Samples/Win32-Http/main.cpp

```diff
@@ -176,9 +176,19 @@ struct SampleHttpCallAsyncContext
     HCCallHandle call;
     bool isJson;
     std::string filePath;
+    std::vector<uint8_t> response;
+    bool isCustom;
 };
 
-void DoHttpCall(std::string url, std::string requestBody, bool isJson, std::string filePath, bool enableGzipCompression, bool enableGzipResponseCompression)
+HRESULT CustomResponseBodyWrite(HCCallHandle call, const uint8_t* source, size_t bytesAvailable, void* context) 
+{
+    SampleHttpCallAsyncContext* customContext = static_cast<SampleHttpCallAsyncContext*> (context);
+    customContext->response.insert(customContext->response.end(), source, source + bytesAvailable);
+    customContext->response.push_back('\0');
```

Comment:
I don't think you want to append a null character each time this method is called.  This may be called several times with a chunk of the response each time, so you'd be inserting some null characters into the middle of the response.  I'd append a null terminator in the HTTP request completion callback below.